Example/src/App.tsx

```diff
@@ -343,4 +344,4 @@ export const styles = StyleSheet.create({
   },
 });
 
-export default App;
+export default gestureHandlerRootHOC(App);
```

Comment:
Personally I think it's more readable to add `<GestureHandlerRootView>` inside `function App` since we already have `<NavigationContainer>` there but I'm also okay with this solution.